src/components/Quizzes/MultipleChoice.jsx

```diff
@@ -13,19 +13,45 @@ import {
 import { useEffect, useState } from "react";
 import { getFirestore, doc, getDoc } from "firebase/firestore";
 import { db } from "../../firebase/config";
+import { Button } from "react-native-paper";
 
 const generateRandomNumber = (min, max) => {
   return Math.floor(Math.random() * (max - min + 1) + min);
 };
 
+
+const ItemComp = (formattedOptns) => {
+  console.log(formattedOptns), '<< ItemComp-formattedOptns'
+  const destructured = formattedOptns
+
+    return (
+      <View style={styles.listItem}>
+        <TouchableOpacity>
+        <Text style={styles.listText}>{formattedOptns.title}</Text>
+
+        </TouchableOpacity>
+      </View>
+
+      
+    )
+
+}
+
+
 export default function MultipleChoice() {
   const [quizData, setQuizData] = useState([]);
   const [selectedValue, setSelectedValue] = useState("");
   const [questionsToCorrectAnswer, setQuestionsToCorrectAnswer] = useState({});
   const [numCorrectAnswers, setNumCorrectAnswers] = useState(0);
   const [LISTDATA, setLISTDATA] = useState([])
+
+  const [currQuestion, setCurrQuestion] = useState([])
+  const [currOptions, setcurrOptions] = useState([])
+
   const [questAnsPair, setQuestAnsPair] = useState({})
+  const [currQName, setCurrQName] = useState([])
   const formatQns = []
```

Comment:
Qns?

Reply:
'Qns' shorthand for 'Questions'. 'formatQns' takes the raw data fetched from API and re-formats to include necessary information for quiz rendering.